src/Policy/OrmResolver.php

```diff
@@ -69,7 +69,8 @@ public function getPolicy($resource)
             return $this->getRepositoryPolicy($resource);
         }
         if ($resource instanceof QueryInterface) {
-            return $this->getRepositoryPolicy($resource->repository());
+            $repository = $resource->getRepository() ?? $resource->repository();
```

Comment:
This check will only work in PHP7.1+. This plugin currently supports PHP 5.6+. You would need to use `method_exists()` to check for the method before calling it instead of a null-coalesce.